printer_mods/hoschi/FYSETC_Spider_DIN_Mount/README.md

```diff
@@ -0,0 +1,43 @@
+FYSETC Spider DIN Rail Mount
+
+for horizontal or vertical mounting
+
+![FYSETC_Spider_DIN_Mount](./IMG/main.jpg)
+
+Required material:
+
+2x printed DIN mounts
+
+for horizontal mounting:
+spider_bracket_horizontal_terminal.stl
```

Comment:
Please add `<br/>` after these lines so they don't appear on a single line.  Alternatively use the list syntax https://www.markdownguide.org/basic-syntax/#unordered-lists